aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -99,181 +106,167 @@ The preceding command adds the [aspnet-codegenerator scaffolding tool](xref:fund
+* In the **Data context class** row, select the **+** (plus) sign.
+  * In the **Add Data Context** dialog, the class name *MvcMovie.Data.MvcMovieContext* is generated, select **Add**.
 
-## Add a database connection string
+![Add Data context](adding-model/_static/dc5.png)
 
-Add a connection string to the *appsettings.json* file:
+* **Views** and **Controller name** : Keep the default.
+* Select **Add**.
 
-# [Visual Studio](#tab/visual-studio)
+### [Visual Studio Code](#tab/visual-studio-code)
 
-[!code-json[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/appsettings.json?highlight=10-12)]
+Open a command window in the project directory. The project directory is the directory that contains the *Program.cs*, *Startup.cs*, and *.csproj* files.
 
-# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
+On Linux, export the scaffold tool path:
 
-[!code-json[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/appsettings_SQLite.json?highlight=10-12)]
-
----
+```console
+export PATH=$HOME/.dotnet/tools:$PATH
+```
 
-Build the project as a check for compiler errors.
+Run the following command:
 
-## Scaffold movie pages
+```dotnetcli
+dotnet-aspnet-codegenerator controller -name MoviesController -m Movie -dc MvcMovieContext --relativeFolderPath Controllers --useDefaultLayout --referenceScriptLibraries
+```
 
-Use the scaffolding tool to produce Create, Read, Update, and Delete (CRUD) pages for the movie model.
+[!INCLUDE [explains scaffold generated params](~/includes/mvc-intro/model4.md)]
 
-# [Visual Studio](#tab/visual-studio)
+### [Visual Studio for Mac](#tab/visual-studio-mac)
 
-In **Solution Explorer**, right-click the *Controllers* folder **> Add > New Scaffolded Item**.
+Open a command window in the project directory. The project directory is the directory that contains the *Program.cs*, *Startup.cs*, and *.csproj* files.
 
-![view of above step](adding-model/_static/add_controller21.png)
+Export the scaffold tool path:
 
-In the **Add Scaffold** dialog, select **MVC Controller with views, using Entity Framework > Add**.
+```console
+export PATH=$HOME/.dotnet/tools:$PATH
+```
 
-![Add Scaffold dialog](adding-model/_static/add_scaffold21.png)
+Run the following command:
 
-Complete the **Add Controller** dialog:
+```dotnetcli
+dotnet-aspnet-codegenerator controller -name MoviesController -m Movie -dc MvcMovieContext --relativeFolderPath Controllers --useDefaultLayout --referenceScriptLibraries
+```
 
-* **Model class:** *Movie (MvcMovie.Models)*
-* **Data context class:** *MvcMovieContext (MvcMovie.Data)*
+[!INCLUDE [explains scaffold generated params](~/includes/mvc-intro/model4.md)]
 
-![Add Data context](adding-model/_static/dc5.png)
+---
 
-* **Views:** Keep the default of each option checked
-* **Controller name:** Keep the default *MoviesController*
-* Select **Add**
+<!-- End of tabs                  -->
 
-Visual Studio creates:
+Scaffolding creates:
```

Comment:
or updates

Reply:
Update the appsettings the connection string. Some place you need to show that.
Update startup and show the changes someplace in this doc.